sql/promscale--0.1.2--0.1.3-beta.sql

```diff
@@ -40,11 +40,11 @@ CREATE AGGREGATE @extschema@.vector_selector(
     sample_time timestamptz,
     sample_value DOUBLE PRECISION)
 (
-    sfunc = vector_selector_transition,
+    sfunc = @extschema@.vector_selector_transition,
```

Comment:
It looks like we have the same pattern of issue in `promscale--0.1.3.beta.sql`

Reply:
What about `extension_sql!` in `lib.rs`? Do we have to fix it there as well, or does it not affect the SQL that ships?

Reply:
That doesn't affect the SQL which ships, so I think that we can skip it for now.